EventGenerator/src/SingleProcessGenerator_module.cc

```diff
@@ -96,14 +101,19 @@ namespace mu2e {
     auto output{std::make_unique<StageParticleCollection>()};
 
     const auto simh = event.getValidHandle<SimParticleCollection>(simsToken_);
-    const auto mus = stoppedMuMinusList(simh);
-
-    for(const auto& mustop: mus) {
+    auto mus_temp = stoppedMuMinusList(simh); //default, negative muon stops
+ 
+    if (pdgId_==-11){
+      mus_temp = stoppedMuPlusList(simh); //positive muon stops
+      muonLifeTime_=0; //decay time already included for stopped muon(+)
```

Comment:
Hi Professor Brown,
I replied this in the email.

Reply:
I think there is still an issue with the stopped mu+ lifetime, but this code functions with data files that currently exist, so it can be used for now.  The deeper issue still needs to be addressed, but outside this PR, and this code will need to be updated once that's fixed.